### DISCUSSION TITLE
SOC with fixed manager IP and dhcp lease for nodes in distributed setup, works then faults

### DISCUSSION THREAD

**Discussion by JLT032** [Security-Onion-Solutions/securityonion#9696] [Unsupported Versions]
Since it seemed a sensible thing to do i rolled out SOC with a manger-VM IP which is fixed and decided to apply dhcp for all nodes.

This works, somehow it takes up to 7 minutes for  a node to update in the grid but it works. Or it worked, after at least more time than the lease time the grid nodes fault and do not recover. I've yet to find a document or command which explains why/how to fix.

To my knowledge the IP did not change for the affected node(s) so I'm eager to learn what may break the grid and/or how to fix it.

JL

**Comment by InfosecGoon** [comment]
So long as the IP is remaining constant, DHCP shouldn't be a problem. That said, a lot of the internode communication is secured with certificates that are pinned to IPs, so if those are changing it will definitely cause problems.

Can you describe your architecture a little more? How many nodes do you have, of what types? Are these on physical hardware, VMs, in the cloud? Do they all fault at the same time or is there a delay? Does so-status show everything up and running on the faulted nodes?

**Comment by JLT032** [comment]
I must confess at this point I'm unable to determine why one interface on the SO VM is unable to receive DHCP leases, it is not specific to SO VM but to that specific interface and DHCP scope. I'm now switching to static IP to just 'get on with it' because this drives me nuts and i don't have the time.

I don't understand the cert-IP issue very well, setting up a cert with SAN should permit to have both IP(s) and hostname(s) present.
